migrations/2025-03-11-015056_local_user_trigger/up.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE
+    local_site
+SET
+    users = users - (
+        SELECT
+            count(*)
+        FROM
+            local_user
+        WHERE
+            NOT accepted_application);
```

Comment:
Instead of subtracting from the current user number I would directly use `count where accepted_application`, in case the current number is wrong.